prober/http_test.go

```diff
@@ -1659,6 +1659,8 @@ func TestSkipResolvePhase(t *testing.T) {
 	}))
 	defer ts.Close()
 
+	// Note: just tested, if local resolve is done or not via metrics. No proxy with diffent resolving available.
```

Comment:
can we reword or remove this comment? I am not sure what this comment is trying to say?

Reply:
Is this better?
`
// Note: just testing via metrics, if local DNS resolve is executed or not. We have in the test setup no proxy with different DNS resolution available.
`